.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -2,10 +2,8 @@
 name: Bug report
 about: Create a report to help us improve
 title: ''
-labels: ''
-assignees: 
-  - vishalnarkhede
-  - madsroskar
+labels: 'Bug'
```

Comment:
Maybe we should add "Needs Triaging" as a default label!!

Reply:
Yeah, I agree. Maybe we should write a short description of the lifecycle of an issue and which labels are used at which stage? Like, needs triaging -> bug / enhancement -> awaiting release and whatever else

Reply:
I second adding "Needs Triaging" as a default label

Reply:
Yes agree!